tasks/php_install.yml

```diff
@@ -1,4 +1,23 @@
 ---
+- name: "[DEBIAN REPO] Debian - repository for php"
+  when: (ansible_facts['distribution'] == "Debian")
+  block:
+    - name: Debian php repository - install required packages
+      ansible.builtin.package:
+        name: gpg
+        state: present
+    - name: Debian php repository - GPG key
+      ansible.builtin.apt_key:
+        url: https://packages.sury.org/php/apt.gpg
+        state: present
+    - name: Debian php repository - APT source
+      ansible.builtin.apt_repository:
+        repo: deb https://packages.sury.org/php/ {{ ansible_distribution_release }} main
+        state: present
```

Comment:
One of the strength of the role is to be customizable in every aspect.
I recommend putting the apt_key url and the apt repository in defaults variable and add some lines about it in the readme
there are many different php repositories that people are using 😃

Reply:
Done :)